lib/testScenario/gen/testScenarioGenerator.ts

```diff
@@ -50,21 +46,21 @@ export interface RequestTracking {
   description: string;
 }
 
-export interface TestScenarioGeneratorOption extends TestResourceLoaderOption {}
+export interface TestScenarioGeneratorOption extends ApiScenarioLoaderOption {}
```

Comment:
This file is used for generate test scenario from payload. I think later we should merge this functionationality with static test scenario generation just in one command.

Reply:
Per discussion offline, we'll do the renaming in this branch.